docs/plus/cronjob.md

```diff
@@ -0,0 +1,93 @@
+# CronJob
+
+CronJob resource is responsible for creating recurring [Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/job/). The job recurrence is determined by a [Schedule]() provided. You can also specify a custom schedule by passing a [cron pattern](https://en.wikipedia.org/wiki/Cron) to Schedule's cron method.
+
+CronJob is similar to a job but it is suitable when there is a need to run a job indefinitely following a schedule. These repitive jobs can be utilized for recurring tasks like backing up a database, pinging a server for health checks, creating snapshots of system and much more.
+
+Some of the properties that can be configured for a CronJob are,
+
+* ### `concurrencyPolicy`
+  Policy to determine how to handle concurrent executions of jobs. The valid values for this policy are, Allow, Forbid and Replace.
+
+* ### `startingDeadline`
+  Deadline for a job to run by if it misses its scheduled time. 
+  
+  For instance, if a job was scheduled to run every 5 minutes and you have mentioned a *startingDeadline* of 30 seconds. Then the job status would be updated as failed if the job does not start by 5 minutes and 30 seconds after the start time of previous job.
+
+* ### `successfulJobsRetained`
+  Number of successful job runs retained during CronJob execution.
+
+* ### `failedJobsRetained`
+  Number of failed job runs retained during CronJob execution.
```

Comment:
This reads very much like an API reference. Remember the added value of a README is that it can be more use-case and intent focused. Instead of repeating the API reference, try formulating these capabilities in terms of use-cases, and provide code snippets for each. 

Also, put this after the basic usage example.

Reply:
I see, makes sense, let me try to rephrase it.